ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -55,6 +63,27 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-aws
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        MAKEFILE_TARGET="create_full_environment run"
+    workflow: baremetalds-assisted
```

Comment:
Please switch to workflow `baremetalds-assisted-workflow` as introduced on https://github.com/openshift/release/pull/15647 to ease the PRs process.